src/bot/commands/stocks.js

```diff
@@ -1,44 +1,138 @@
-import { RichEmbed } from 'discord.js';
+import debug from 'debug';
+import { RichEmbed, Permissions } from 'discord.js';
+import fetch from 'node-fetch';
+import { fromUnixTime, format } from 'date-fns';
+
+const log = debug('stocks');
 
 // TODO: Write Info
 export const info = {
   name: 'stocks',
-  description: '',
-  usage: '',
-  disabled: true,
+  description: 'Get some handy dandy stock info for your day trading!',
+  permissions: [Permissions.FLAGS.EMBED_LINKS],
+  usage: '```@Aquarius stocks <stock symbol>```',
 };
+function numberWithCommas(x) {
+  return x.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ",");
+}
+
+const API = 'https://cloud.iexapis.com/stable/stock/';
+
+const TOKEN = `quote?token=${process.env.IEXCLOUD_KEY}`;
 
-async function getStock(sign) {
-  return Promise.resolve(sign);
+function createStocksEmbed(data) {
+  const change = `Change ${
+    data.change >= 0
+      ? ':chart_with_upwards_trend:'
+      : ':chart_with_downwards_trend:'
+  }`;
+  const embed = new RichEmbed({
+    description: `*Stock Symbol: $${data.symbol}*`,
+    color: 16318579,
+    timestamp: `${fromUnixTime(data.latestUpdate / 1000)}`,
```

Comment:
Don't think we need the string interpolation here (and below in some places) just need `fromUnixTime(data.latestUpdate / 1000)`